src/main/java/com/icthh/xm/tmf/ms/communication/dto/InlineKeyboardMarkupDto.java

```diff
@@ -0,0 +1,27 @@
+package com.icthh.xm.tmf.ms.communication.dto;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.pengrad.telegrambot.model.request.InlineKeyboardButton;
+import com.pengrad.telegrambot.model.request.InlineKeyboardMarkup;
+import com.pengrad.telegrambot.model.request.Keyboard;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@JsonTypeName("inline")
+public class InlineKeyboardMarkupDto extends KeyboardDto {
+
+    private List<List<InlineKeyboardButtonDto>> inlineKeyboard;
+
+    @Override
+    public Keyboard build() {
+        List<List<InlineKeyboardButtonDto>> inlineKeyboard = nullSafe(this.inlineKeyboard);
+        InlineKeyboardButton[][] keyboardMarkup = nullSafe(inlineKeyboard).stream()
+                .map(InlineKeyboardButtonDto::build)
+                .map(l -> nullSafe(l).toArray(new InlineKeyboardButton[0]))
```

Comment:
Suggest considering to add a helper method to `KeyboardDto`:
```
 @SuppressWarnings("unchecked")
    protected <T> T[] toArray(List<T> list){
        return list.toArray((T[]) new Object[0]);
    }
```

it will allow to make lambda cleaner: 
```
KeyboardButton[][] keyboardMarkup = nullSafe(inlineKeyboard)
            .stream()
            .map(KeyboardButtonDto::build)
            .map(this::nullSafe)
            .map(this::toArray)
            .toArray(KeyboardButton[][]::new);
```